code/modules/clothing/accessories/holster.dm

```diff
@@ -49,7 +49,8 @@
 				SPAN_NOTICE("[user] draws \the [holstered], pointing it at the ground."),
 				SPAN_NOTICE("You draw \the [holstered], pointing it at the ground.")
 				)
-		user.put_in_active_hand(holstered)
+		if(!user.put_in_active_hand(holstered)) //Occulus Edit: If your primary hand is full, draw with your offhand
+			user.put_in_inactive_hand(holstered) //Occulus Edit: Prevents guns from getting deleted with hotkeys.
```

Comment:
`put_in_hands(holstered)`
also bap, atomization >:C
also, just noticed this says "you draw the gun" even if you can't draw the gun (if hands full)

Reply:
if(istype(user.get_active_hand(),/obj) && istype(user.get_inactive_hand(),/obj))
		to_chat(user, SPAN_WARNING("You need an empty hand to draw \the [holstered]!"))|

Check above